README.md

```diff
@@ -62,30 +83,6 @@ Add the `help` flag on any command to see how you can use it. For example, `yarn
 
 The `yarn run` command will list all of the scripts available to run for this project.
```

Comment:
Pls make it clear where to run yarn or mvn when we describing about uaa or gateway